qemu/tests/blockdev_mirror_cancel_running_job.py

```diff
@@ -8,7 +8,7 @@ class BlockdevMirrorCancelRunningJob(BlockdevMirrorNowaitTest):
     """
 
     def cancel_job(self):
-        self.main_vm.monitor.cmd("job-cancel", {'id': self._jobs[0]})
+        self.main_vm.monitor.cmd("block-job-cancel", {'device': self._jobs[0]})
```

Comment:
Do we also need 'force: True' here? I checked the qmp doc, but didn't find the default value for force

Reply:
This params is neccessary only when cancelling a mirror job who reach "ready" status. In my case, I cancel the job just after mirror job started.
So I will update my case to add a random sleep time before mirror job cancel and then add this param, thanks zhenchao.

Reply:
Sorry zhenchao, need to update the comment again. In fact, we have covered the test senario for "cancel a mirror job that in ready status" with script "blockdev_mirror_cancel_ready_job_with_io", so in this script "blockdev_mirror_cancel_running_job", cover scenario "cancel a mirror job before it reachesready status" is enough. So here, param "force=ture" is not needed.